notebook/notebookapp.py

```diff
@@ -238,6 +238,20 @@ def init_handlers(self, settings):
         handlers.extend(load_handlers('services.nbconvert.handlers'))
         handlers.extend(load_handlers('services.kernelspecs.handlers'))
         handlers.extend(load_handlers('services.security.handlers'))
+        
+        # BEGIN HARDCODED WIDGETS HACK
+        try:
+            import ipywidgets
+            handlers.append(
+                (r"/nbextensions/widgets/(.*)", FileFindHandler, {
+                    'path': ipywidgets.find_static_assets(),
+                    'no_cache_paths': ['/'], # don't cache anything in nbextensions
```

Comment:
Too late for that https://github.com/ipython/ipykernel/pull/19 .  This is just a temporary thing, until the packaging problem is solved.